src/smart-components/workspaces/overview/about-access-tab.tsx

```diff
@@ -0,0 +1,254 @@
+import React from 'react';
+import { ContentHeader, ServiceCard } from '@patternfly/react-component-groups';
+import {
+  Button,
+  ButtonVariant,
+  Card,
+  CardBody,
+  CardFooter,
+  CardTitle,
+  DataList,
+  ExpandableSection,
+  Gallery,
+  GalleryItem,
+  Label,
+  List,
+  ListItem,
+  PageSection,
+  Text,
+  TextVariants,
+  Title,
+} from '@patternfly/react-core';
+import CustomDataListItem, { IconName } from './custom-data-list-item';
+import { ExternalLinkAltIcon, InfrastructureIcon, UsersIcon } from '@patternfly/react-icons';
+import { Table, Tbody, Tr, Td } from '@patternfly/react-table';
+
+const VIEW_DEFAULT_GROUPS = 'https://console.redhat.com/iam/user-access/groups';
+// to do - update link when available
+const GRANT_ACCESS = '';
+
+const WorkspacesOverview = () => {
+  const workspacesIcon = '/apps/frontend-assets/rbac-landing/rbac-landing-icon.svg';
+  const bindingsIcon = '/apps/frontend-assets/rbac-landing/workspaces-bindings-icon.svg';
+  const rolesIcon = '/apps/frontend-assets/rbac-landing/workspaces-roles-icon.svg';
```

Comment:
It is generally a better practice to move these constants above the component so they are not recreated on every render